framework/audio/engine/iaudioengine.h

```diff
@@ -56,6 +57,8 @@ class IAudioEngine : MODULE_EXPORT_INTERFACE
     virtual void setVolume(handle h, float volume) = 0; // 0. - 1.
     virtual void setPan(handle h, float val) = 0; // -1 only left, 0 center, 1 only right
     virtual void setPlaySpeed(handle h, float speed) = 0;
+
+    virtual async::Notification playCallbackCalled() const = 0; //! NOTE A portion of data was given to the driver
```

Comment:
What is the purpose of this notification?

Reply:
Calling this callback means that the play position has changed. I planing that on this call will determine which events (notes) are currently being played.